osu.Game/Rulesets/Edit/DistancedHitObjectComposer.cs

```diff
@@ -168,5 +181,26 @@ public ExpandingToolboxContainer()
                 FillFlow.Spacing = new Vector2(10);
             }
         }
+
+        private class DistanceSpacingToast : Toast
+        {
+            private readonly ValueChangedEvent<double> change;
+
+            public DistanceSpacingToast(LocalisableString value, ValueChangedEvent<double> change)
+                : base(getAction(change).GetLocalisableDescription(), value, string.Empty)
+            {
+                this.change = change;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OsuConfigManager config)
+            {
+                ShortcutText.Text = config.LookupKeyBindings(getAction(change));
```

Comment:
This is pretty bad, but similar to what we used in `MusicKeyBindingHandler` as well. Open to moving that to a better method, not sure where it should be placed though. `OsuGame`? or maybe that method shouldn't exist in the first place and let each usage check `RealmKeyBindingStore.GetReadableCombination`.

Reply:
Not sure what's so bad about this? I don't see the issue.

I do see an issue however in that `MusicKeyBindingHandler` generally displays the binding uppercased:

```suggestion
                ShortcutText.Text = config.LookupKeyBindings(getAction(change)).ToUpper();
```

Reply:
Bad is that it resolves `OsuConfigManager` to look up for key binding strings, the operation is completely irrelevant from the class.

Reply:
I'd rather have this kind of glue code duplicated than call some abstract monstrosity into being, quite frankly. It's like three lines.